brownie/project/main.py

```diff
@@ -588,7 +588,8 @@ def from_brownie_mix(
     Returns the path to the project as a string.
     """
     project_name = str(project_name).lower().replace("-mix", "")
-    url = MIXES_URL.format(project_name)
+    default_branch = _get_mix_default_branch(project_name)
+    url = MIXES_URL.format(project_name, default_branch)
```

Comment:
Line 601: ` project_path.parent.joinpath(project_name + "-mix-master").rename(project_path)`

Here the reference to master needs to be changed too.

Reply:
Fixed with 2e16fa5